src/modules/home/components/card-recommended/CardRecommended.vue

```diff
@@ -46,17 +54,33 @@
   </div>
 </template>
 
-<script>
+<script lang="ts">
 import { defineComponent } from "vue";
 import BaseButton from "@/commons/UI/BaseButton.vue";
 import ArrowRightIcon from "@/assets/ArrowRight.vue";
 import useCardRecommended from "./useCardRecommended";
+import { Event } from "@/commons/Interfaces/index.ts";
+import LazyImage from "@/commons/UI/lazy-image/LazyImage.vue";
 
 export default defineComponent({
   components: {
+    LazyImage,
     BaseButton,
     ArrowRightIcon
   },
+  props: {
+    recommended: {
+      type: Object as () => Event,
+      default: {
+        day: 99,
```

Comment:
I don't think we need default props, since the component will not be loaded anyway if there is no data.